apps/server/src/modules/video-conference/uc/video-conference-create.uc.ts

```diff
@@ -38,6 +38,12 @@ export class VideoConferenceCreateUc {
 	}
 
 	private async create(currentUserId: EntityId, scope: ScopeRef, options: VideoConferenceOptions): Promise<void> {
+		/* need to be replace with
```

Comment:
Willst du das noch in diesem PR machen?

Reply:
Nein die Kommentare in tools und BBB sind für capcakes.